### PR TITLE
fix lint errors

### PR DESCRIPTION
**Pull request by mfdeveloper508**

### CODE REVIEW

File: pkg/util/util.go [hunk 1]
```diff
@@ -111,23 +111,22 @@ func ValidateField(data any, spec rules.ElementRule, mask string) error {
...
-			// TODO
-			fmt.Errorf("the element is not used")
-		}
-		break
+	/*
+		case rules.MASK_REQUIRED:
+			if dataStr == "" {
+				// TODO
+				return errors.New("the element is require")
+			}
```
Comment (adamdecaf) [inline pkg/util/util.go:119]: Why comment this out? Removing `break` is all that's needed.

File: pkg/loops/loop.go [hunk 2]
```diff
@@ -112,12 +112,6 @@ func (r *Loop) Parse(data string, args ...string) (int, error) {
 	index := 0
 
 	for rule := segRules.Get(index); rule != nil; rule = segRules.Get(index) {
-
-		cnt := 1
-		if rule.RepeatCount > cnt {
-			cnt = rule.RepeatCount
-		}
```
Comment (adamdecaf) [inline pkg/loops/loop.go:119]: Shouldn't `cnt` still be assigned to `rule.RepeatCount`?

Comment (mfdeveloper508) [reply pkg/loops/loop.go:119]: Repeat() function will return 1 if RepeatCount is not specified

File: pkg/util/util.go (hunk 1 above)
Comment (mfdeveloper508) [reply pkg/util/util.go:119]: we need consider the code carefully.
to be honest, there isn't any describe for this logic, so I want to fix this code after discuss real use cases with customers

File: pkg/loops/loop.go (hunk 2 above)
Comment (mfdeveloper508) [reply pkg/loops/loop.go:119]: if RepeatCount is omit, Repeat() functions will return default value 1

File: pkg/util/util.go (hunk 1 above)
Comment (adamdecaf) [reply pkg/util/util.go:119]: Sounds good to me.

File: pkg/loops/loop.go (hunk 2 above)
Comment (adamdecaf) [reply pkg/loops/loop.go:119]: Ah okay.

**Pull request merged**